assets/templates/index.html

```diff
@@ -244,7 +244,7 @@ <h2>Botones</h2>
             </div>
           </div><!-- button-card -->
           <div class="buttons-card">
-            <button type="button" class="dp-button button-medium primary-green" disabled>Button Disabled</button>
+            <button type="button" class="dp-button button-medium primary-green button--loading" disabled>Button Disabled</button>
```

Comment:
Is it the right element to demonstrate this class?

Reply:
Solo queria ver como se comportaba con el disabled, nada importante

Reply:
Que opinás de agregarle disabled al botón "spinner" que está un poco más arriba?

```
<button type="button" disabled class="dp-button button-medium primary-green button--loading">Spinner</button>
                      ^^^^^^^^
```

Al fin y al cabo, en general cuando tenga el estado de _loading_, el botón va a estar deshabiltado.